cmd/devstream/delete.go

```diff
@@ -18,6 +19,10 @@ DevStream will delete everything defined in the config file, regardless of the s
 
 func deleteCMDFunc(cmd *cobra.Command, args []string) {
 	log.Println("Delete started.")
-	pluginengine.Delete(configFile)
+	err := pluginengine.Delete(configFile)
+	if err != nil {
```

Comment:
maybe the following format is better: 
if err := func(); err != nil {
 //todo
}